actions/workflows/st2_prep_release.yaml

```diff
@@ -125,6 +125,16 @@ st2cd.st2_prep_release:
                 local_repo: <% 'st2_dockerfiles_' + $.local_repo_sfx %>
                 hosts: <% $.host %>
                 cwd: <% $.cwd %>
+            on-success:
+                - wait_for_community_docker_images
+        wait_for_community_docker_image:
+            action: st2cd.wait_for_docker_image
+            input:
+                org: <% $.fork %>
+                image: st2workflowengine
```

Comment:
First versions of the code used `st2`, but then I figured since that's not the last image pushed to docker hub we could get into a case where the required image isn't available. However, on further reflection, I'll use `st2web`, since it's the last required image pushed to docker hub.

Reply:
That makes sense :+1: